Integration/Integration.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,1379 @@
+		};
+		0A87D4332175E84100BD0C4C /* Main.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				0A87D4342175E84100BD0C4C /* Base */,
+			);
+			name = Main.storyboard;
+			sourceTree = "<group>";
+		};
+		0A87D4432175E93400BD0C4C /* Interface.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				0A87D4442175E93400BD0C4C /* Base */,
+			);
+			name = Interface.storyboard;
+			sourceTree = "<group>";
+		};
+		0A87D5492177C22600BD0C4C /* Main.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				0A87D54A2177C22600BD0C4C /* Base */,
+			);
+			name = Main.storyboard;
+			sourceTree = "<group>";
+		};
+		0A87D54E2177C22700BD0C4C /* LaunchScreen.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				0A87D54F2177C22700BD0C4C /* Base */,
+			);
+			name = LaunchScreen.storyboard;
+			sourceTree = "<group>";
+		};
+		0A87D5592177C25700BD0C4C /* Interface.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				0A87D55A2177C25700BD0C4C /* Base */,
+			);
+			name = Interface.storyboard;
+			sourceTree = "<group>";
+		};
+/* End PBXVariantGroup section */
+
+/* Begin XCBuildConfiguration section */
+		0A87D3642175E4A300BD0C4C /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 0A87D4DB2177345B00BD0C4C /* Module-Debug.xcconfig */;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
+				CLANG_WARN_BOOL_CONVERSION = YES;
+				CLANG_WARN_COMMA = YES;
+				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_EMPTY_BODY = YES;
+				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
+				CLANG_WARN_INT_CONVERSION = YES;
+				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
+				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				COPY_PHASE_STRIP = NO;
+				CURRENT_PROJECT_VERSION = 1;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_TESTABILITY = YES;
+				GCC_C_LANGUAGE_STANDARD = gnu11;
+				GCC_DYNAMIC_NO_PIC = NO;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREPROCESSOR_DEFINITIONS = (
```

Comment:
I fear we're missing out on build settings here. E.g. for Swift projects, there's the `DD_LOG_LEVEL` definition, that was added in #952, that we should add.

Reply:
@ffried glad you brought this up. What happens if there is not preprocessor definition with `DD_LOG_LEVEL`?

Reply:
Nothing really. There's a default `define` to `DDLogLevelAll` [here](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/1723be8823ea5c73fb69a30d8fca3edf64b48daa/Classes/SwiftLogLevel.h#L13).

Reply:
But we should still have it in our examples / tests.